README.md

```diff
@@ -35,6 +35,10 @@ or `skopeo`:
   * `quay.io/containers/*:latest` and `quay.io/*/stable:latest` -
     Built daily using the same `Containerfile` as above.  The tool versions
     will remain the "latest" available in Fedora.
+  * `quay.io/containers/aio:latest` and `quay.io/containers/aio:<date stamp>` -
```

Comment:
```suggestion
  *[ `quay.io/containers/aio:latest`](aio/README.md) and [`quay.io/containers/aio:<date stamp>`](aio/README.md) -
```
See readme below for context, and I'm not 100% sure of my syntax here.

Reply:
I'll add a link at the end of the file, right after the related P, B, and S links.